pkg/policy/policy_file_test.go

```diff
@@ -188,15 +195,177 @@ func TestPolicyValidate(t *testing.T) {
 			},
 			expectedError: errors.New("policy.validateAction: policy invalid_policy_action, action audit is not valid"),
 		},
-		// invalid args?
-		// invalid retval?
+		{
+			testName: "invalid retval",
+			policy: PolicyFile{
+				Name:          "invalid_retval",
+				Description:   "invalid retval filter",
+				Scope:         []string{"global"},
+				DefaultAction: "log",
+				Rules: []Rule{
+					{
+						Event: "write",
+						Filter: []string{
+							"retval",
+						},
+					},
+				},
+			},
+			expectedError: errors.New("policy.PolicyFile.validateRules: policy invalid_retval, invalid filter operator: retval"),
```

Comment:
Just a note for ourselves of the future: We should use error variables to be possible to compare them in tests and further code path. I've put a comment about this in https://github.com/aquasecurity/tracee/pull/3137/files#diff-f70250a062f09eebb2d5b94837022723ff90b7175790d0d7326da75f1ade0222R360-R361

The unique concern is how to make it comparable when the error is a formatted one.